javascript-forms/form-events.js

```diff
@@ -0,0 +1,31 @@
+var $formControlUserName = document.forms[0].elements.name;
+
+var $formControlUserEmail = document.forms[0].elements.email;
+
+var $formControlMessage = document.forms[0].elements.message;
```

Comment:
What's this for?

Reply:
Instead of using `document.querySelector()`, I just used the accessed the forms collection and the elements collection as noted on page 572 of the Duckett book to make sure it (mostly) made sense to me. I could have used the `document.querySelector()` instead.

Reply:
It's how I interpreted #4 and #5 of the exercise.